content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation][2] is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries][3] from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Prerequisites
+
+Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+
+<div class="alert alert-warning">
+When replacing an existing Datadog instrumentation with its OpenTelemetry equivalent, remember to disable the
+Datadog instrumentation to avoid duplicate spans in the trace.
+</div>
+
+## Language support
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Python   | v2.10.0                  |
+| Ruby     | v2.1.0                   |
+| Go       | -                        |
```

Comment:
Is there a reason we are not calling out min languages for these four?

Reply:
Munir helped fill in some of the versions, the rest were going to be filled in as their tabs were added (the idea was to merge this first, to make it easier for each language to work on a separate follow-up PR rather than have this PR as a bottleneck)

Reply:
OK, version table is now complete - thanks to everyone who contributed!